canonical/cloud-init
# remove unnecessary reboot from gpart resize

## Proposed Commit Message

> remove unnecessary reboot from gpart resize
>
> a reboot after `gpart resize` hasn't been necessary for a long while!


## Additional Context

ref https://wiki.freebsd.org/GrowingFilesystemsOnline
LP# 1902370

## Test Steps

- resize the server's disk image (in my case this was done with qemu-img resize while the server was off, i think that's probably not necessary) 
- start server up
- `gpart list` should show the partitions as it "corrupt"
- run `gpart recover…` as per our code
- run `gpart resize…` as per our code
- `gpart list` should  now show the new size
- run `zpool online…` as per our code
- `zfs list` should show the new size

## Checklist:

 - [ ] I haven't updated or added any unit tests accordingly, because there's no tests right now, and I'm not sure I wanna be the first…
